roseau/load_flow/models/lines/parameters.py

```diff
@@ -103,7 +103,6 @@ def with_shunt(self) -> bool:
     @ureg_wraps(
         None,
         (
-            None,
             None,
             None,
             "ohm/km",
```

Comment:
You may be able to collapse this to a single line now if you remove the trailing comma